homeassistant/components/nextcloud/entity.py

```diff
@@ -21,7 +20,7 @@ def __init__(
         """Initialize the Nextcloud sensor."""
         super().__init__(coordinator)
         self.item = item
-        self._attr_name = item
+        self._attr_translation_key = slugify(item)
```

Comment:
Why not just lower case this instead of using slug?

Reply:
Some strings contains periods, and our scripts don't like periods in translationkeys